test/cypress/features/login.feature

```diff
@@ -56,4 +56,15 @@ Feature: Login
     When I am on dashboard page
     Then I should be connected to network devnet
 
+  Scenario: Log out
+    Given showNetwork setting is true
+    Given I am on Login page
+    When I choose testnet
+    When I enter first passphrase of testnet_guy
+    When I login
+    When I am on dashboard page
+    And I change active token to BTC
+    And I log out
+    Then I should see lisk monitor features
```

Comment:
E2E tests are slow and Logging in a long process. Please consider removing this scenario and simply adding the last three steps to one of the existing scenarios.